nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/authorization/AuthorizeControllerServiceReference.java

```diff
@@ -162,7 +162,6 @@ private static void authorize(Authorizer authorizer, AuthorizableLookup lookup,
             Authorizable serviceAuthorizable = service.getAuthorizable();
 
             serviceAuthorizable.authorize(authorizer, RequestAction.READ, user);
-            serviceAuthorizable.authorize(authorizer, RequestAction.WRITE, user);
```

Comment:
@markap14 this change is basically a fix for https://issues.apache.org/jira/browse/NIFI-9895. I thought hard and this change looks reasonable for me. But maybe I'm missing something. Could there be a reason to allow the change of a controller service reference only to users who have write access to that controller service - when it's referenced via a parameter context?